TestFoundation/xdgTestHelper/main.swift

```diff
@@ -7,10 +7,10 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
-#if DEPLOYMENT_RUNTIME_OBJC || os(Linux) || os(Android)
-import Foundation
-#else
+#if canImport(SwiftFoundation)
```

Comment:
Can all the `canImport…` be turned into `canImport(…) && !DEPLOYMENT_RUNTIME_OBJC`? This allows DarwinCompatibilityTests to keep working.

Reply:
Sure, if I can just point out that this is exactly the problem with `canImport` :-p